grammars/less.cson

```diff
@@ -457,7 +457,7 @@
             'include': '#strings'
           }
           {
-            'match': '(\\b|\\.{0,2}/).*\\b'
+            'match': '(\\b|\\.{0,2}/)\\S*\\b'
```

Comment:
Would a better fix be to stop matching once a closing parenthesis is encountered?